### PR TITLE
Ui refresh lms integration

### PR DESCRIPTION
**Pull request by Wardormeur**
Reorganize LMS inte for new UI
duplicate of https://github.com/CoderDojo/cp-zen-platform/pull/929

### CODE REVIEW

File: lib/profiles.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ exports.register = function (server, options, next) {
     path: options.basePath + '/profiles/create',
     handler: handlers.actHandlerNeedsUser('create'),
     config: {
-      description: 'Create a user profile',
+      description: 'Create/update an user profile',
```
Comment (DanielBrierton) [inline lib/profiles.js]: "a user"

File: web/public/js/directives/cd-connect-lms/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            });
+          };
+
+          $scope.getLoginURL = function () {
+            cdLMSService.getLoginURL({}, callback);
+
+            function callback (link) {
+              if (link.approvalRequired){
+                alertService.confirm(
+                  $translate.instant('By using this functionnality, you allow us to share basic information with this provider (email, name, user type (Parent, Mentor, Champion))\n Do you agree?'),
```
Comment (DanielBrierton) [inline web/public/js/directives/cd-connect-lms/index.js]: *functionality

Also, this string isn't in the messages.po

I'd also be inclined to not put brackets within brackets.

Comment (Wardormeur) [inline web/public/js/directives/cd-connect-lms/index.js]: I simply delete the details of UserType ?

Comment (DanielBrierton) [inline web/public/js/directives/cd-connect-lms/index.js]: Yeah, I think so. Seems unnecessary to list them out imo.

File: web/locale/en_US/messages.po [hunk 3]
```diff
@@ -3035,8 +3035,14 @@ msgstr " and see what we are working on, as well as the latest features for Zen.
...
 
-msgid "By using this functionality, you allow us to share basic information with this provider (email, name, user type)\n Do you agree?"
-msgstr "By using this functionality, you allow us to share basic information with this provider (email, name, user type)\n Do you agree?"
+msgid "By using this functionality, you allow us to share basic information with this provider (email, name, user type)"
+msgstr "By using this functionality, you allow us to share basic information with this provider (email, name, user type)"
+
+msgid "Only modules for mentors and champions are available at the moment, come back later on to discover more!"
+msgstr "Only modules for mentors and champions are available at the moment, come back later on to discover more!"
+
+msgid "Do you agree ?"
```
Comment (DanielBrierton) [inline web/locale/en_US/messages.po]: Shouldn't be a space before the ?

**Pull request merged**